compose/snippets/src/main/java/com/example/compose/snippets/layouts/PagerSnippets.kt

```diff
@@ -426,7 +426,7 @@ private fun CustomSnapDistance() {
         HorizontalPager(
             state = pagerState,
             pageSize = PageSize.Fixed(200.dp),
-            beyondBoundsPageCount = 10,
+            outOfBoundsPageCount = 10,
```

Comment:
is this change reflected in the guidance as well?

Reply:
This change won't be affected in the guidance on DAC since it's on the `latest` branch. However, we'll need to update this snippet once 1.7.0 is stable (we'll have to do that on the `main` branch)